crates/dbsp/src/utils/tuple/gen.rs

```diff
@@ -75,7 +75,6 @@ macro_rules! declare_tuples {
                 }
             }
 
-            /*#[cfg(test)]
             impl<$($element),*> ToSqlRow for $tuple_name<$($element,)*>
```

Comment:
how about instead of moving this all into dbsp, we make a macro that *just* implements ToSqlRow for Tup1-10 (and we call that where the ToSqlRow trait was before (similarly to how we do it with the serde trait) -- and move the trait back there) and then you invoke the extra macro (the one that implements ToSqlRow) whenever you instantiate a tuple >10. I'll make those changes since I clearly broke it by uncommenting ToSqlRow...

Reply:
This is how I tried initially, but I failed.